appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.mapper.js

```diff
@@ -207,63 +207,74 @@ function generateCommentSummaryList(comment) {
 			  )
 			: null;
 
+	const rows = [
+		{
+			key: { text: 'Interested party' },
+			value: { text: comment.represented.name }
+		},
+		{
+			key: { text: 'Email' },
+			value: { text: comment.represented.email }
+		},
+		{
+			key: { text: 'Address' },
+			value: hasAddress
+				? { html: addressToMultilineStringHtml(comment.represented.address) }
+				: { text: 'Not provided' },
+			actions: {
+				items: [
+					hasAddress
+						? { text: 'Change', href: '#', visuallyHiddenText: 'address' }
+						: { text: 'Add', href: '#', visuallyHiddenText: 'address' }
+				]
+			}
+		},
+		{
+			key: { text: 'Submitted' },
+			value: { html: dateISOStringToDisplayDate(comment.created) }
+		},
+		{
+			key: { text: comment.redactedRepresentation ? 'Original comment' : 'Comment' },
+			value: {
+				text: commentIsDocument ? 'Added as a document' : comment.originalRepresentation
+			},
+			actions: {
+				items: commentIsDocument ? [] : [{ text: 'Redact', href: '#' }]
+			}
+		},
+		...(comment.redactedRepresentation
+			? [{ key: { text: 'Redacted comment' }, value: { text: comment.redactedRepresentation } }]
+			: []),
+		{
+			key: { text: 'Supporting documents' },
+			value: attachmentsList ? { html: attachmentsList } : { text: 'Not provided' },
+			actions: {
+				items: [
+					...(comment.attachments?.length > 0
+						? [{ text: 'Manage', href: '#', visuallyHiddenText: 'supporting documents' }]
+						: []),
+					{ text: 'Add', href: '#', visuallyHiddenText: 'supporting documents' }
+				]
+			}
+		}
+	];
+
+	if (!isReviewPage) {
+		rows.splice(3, 0, {
+			key: { text: 'Site visit requested' },
+			value: { text: comment.siteVisitRequested ? 'Yes' : 'No' }
+		});
+	}
+
+	if (!isReviewPage && comment.status === COMMENT_STATUS.INVALID) {
+		rows.push({
```

Comment:
Again you could do this with conditional destructuring, just to keep things consistent, but there's nothing wrong with this approach